GianCHerrera/SimetrikTechInterview
# Web Automation - Menu Validations

## Description

This pull request includes the following changes:
- Implemented scenarios to automate test cases related to validate menu options.
- Added feature file named `techInterview.feature`

## Execution Instructions

To test these changes:
1. Clone this repository and switch to the feature/techInterview branch.
2. Install packages using the following command.
`python3 -m pip install requirements.txt`
4. Currently, the created scenarios are tagged as @TechInterview.
6. Run the following command in the terminal so recently added scenarios will run:
` behave --no-capture --format plain --tags=@TechInterview -Dcountry=co -Dtestrail=false -Ddriver=local`

:warning: **Warning:** Make sure you are using Python 3.9

## Additional Notes

- A total of 2 scenarios were added where 1 scenario is a Scenario Outline including 9 Examples.

## Successful Execution Tests
![image](https://github.com/GianCHerrera/SimetrikTechInterview/assets/83032013/aba1144a-5ae8-4e45-aa1b-a97505f56c0d)



### Author:
@GianCHerrera
